package.json

```
@@ -0,0 +1,8 @@
+{
+    "name": "maeparser",
+    "version": "1.2.4",
```

Based on the key listed in https://flaviocopes.com/package-json, maybe add:

```suggestion
    "author": "schrodinger",
    "repository": "github:schrodinger/maeparser",
    "license": "MIT",
    "version": "1.2.4",
```